frontend/src/components/Overview.module.scss

```
@@ -134,5 +134,11 @@
       @extend .layout-row;
       gap: 10px;
     }
+    a {
+      overflow-wrap: normal;
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Extending existing styles seems to also do the trick
```suggestion
      @extend .hide-text-ellipsis;
```